vtex-apps/store-components
# Adding prop to remove overlay

#### What problem is this solving?
This solves the overlap problem, because currently when we set the option isOpen true, the suggestions override all other content


#### How to test it?

first step click on search field 
![image](https://user-images.githubusercontent.com/38354801/100007279-9b696d00-2daa-11eb-8926-d19e4bd371fd.png)

later change props on block to see behavior

![image](https://user-images.githubusercontent.com/38354801/100007658-2480a400-2dab-11eb-9f3a-ad6ecee9c2c2.png)



[Workspace](https://mcdon28--devmcdbrdeliveryio.myvtex.com/)

#### Screenshots or example usage:



Before: 
![image](https://user-images.githubusercontent.com/38354801/100007932-8b9e5880-2dab-11eb-8062-030bdb5e0b07.png)

After: 
![image](https://user-images.githubusercontent.com/38354801/100008141-ca341300-2dab-11eb-8be8-fffe402af918.png)



#### How does this PR make you feel? [:link:](http://giphy.com/)

Very happy to be able to contribute with vtex


![coding](https://media.giphy.com/media/PiQejEf31116URju4V/giphy.gif)
